worksheet_B/Mandelbrot.cpp

```diff
@@ -2,6 +2,84 @@
 //
 
 #include "stdafx.h"
+#include <vector>
+
+/*
+Struct definitions and hsv2rgb() function below courtesy of 
+http://stackoverflow.com/questions/3018313/algorithm-to-convert-rgb-to-hsv-and-hsv-to-rgb-in-range-0-255-for-both
```

Comment:
Quite a lot of code here -- might have been better to split it into a separate file